docs/Conformance.md

```diff
@@ -54,7 +56,7 @@ FHIR search modifiers are described at https://www.hl7.org/fhir/R4/search.html#m
 |--------------------------|-------------------|-----------------------------------------------------|
 |String                 |`:exact`,`:contains`,`:missing` |Performs a "starts with" search that is case-insensitive and accent-insensitive|
 |Reference              |`:[type]`,`:missing`            |Performs an exact match search|
-|URI                    |`:below`,`:missing`             |Performs a "starts with" search|
+|URI                    |`:below`,`:missing`             |Performs a "starts with" search (issue #273)|
```

Comment:
do we really need the issue number here?

Reply:
the intention is to help us remember to update it as part of #273 or, if we forget, for folks to know its stale when they see #273 is resolved